packages/building-blocks/src/stories/alerts.story.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ReactElement } from "react";
+import { Meta } from "@storybook/react";
+import { ExclamationTriangle } from "@styled-icons/bootstrap/ExclamationTriangle";
+import Alert from "../alert/Alert";
+import red from "../colors/red";
+
+const meta: Meta<typeof Alert> = {
+  title: "Building-Blocks/Alert",
+  component: Alert
+};
+
+export default meta;
+
+const alerts = [
+  {
+    alertUrl: "http://trimet.org/alerts/",
+    effectiveStartDate: 1576471800,
+    alertDescriptionText:
+      "TriMet Customer Service will be unavailable to serve text messages or Twitter responses from 9:00 p.m.- 11:30 p.m. For immediate assistance regarding safety or security concerns, please contact the police via 911."
+  },
+  {
+    alertUrl:
+      "https://news.trimet.org/2019/11/next-up-for-elevator-improvements-sunset-transit-center-park-ride/",
+    effectiveStartDate: 1573083439,
+    alertDescriptionText:
+      "The Park and Ride garage elevator at Sunset Transit Center is closed for approximately 3 months for improvements. During this time garage users must use the stairs or find alternate parking. Visit trimet.org/parkandride for a complete list of Park and Ride garages."
+  },
+  {
+    alertUrl: "http://trimet.org/alerts/",
+    effectiveStartDate: 1572827580,
+    alertDescriptionText:
+      "The west elevators at the Washington Park MAX Station are out of service. Please use east elevators to access street level and platforms. "
+  }
+];
+
+const alertKey = crypto.randomUUID();
```

Comment:
Why not just use `i` for the key? Does that break something?

Reply:
actually yeah this is bad because all the keys will all be the same!

Reply:
Good catch!! I moved this into the `map` so it actually does what it's supposed to do, although I guess `i` would work in this case because the order of the mock alerts in this story isn't going to change? Happy to change it if someone feels strongly?

Reply:
Actually I didn't realize this is just the story. It's fine, stories don't have to be perfect!